### PR TITLE
marino/pxp-223-readme-update updating new format and readme for example1

### PR DESCRIPTION
**Pull request by Marinolinderhof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
+- [Styling](https://docs.mollie.com/guides/mollie-components/styling)
 
-## Example 1 (regular labels)
-[HTML](example-1/index.html) | [CSS](example-1/style.css) | [Javascript](example-1/script.js)
+## Examples
 
-![](screenshots/example-1.gif)
+- [Example 1](./example-1) Bare minimum example ([codepen](example-2))
+- [Example 2](./example-2) Floating labels ([codepen](example-2))
+- [Example 3](./example-3) Compact version ([codepen](example-2))
```
Comment (pabloroman) [inline README.md]: I think you are aware but the link to Example 3 and the three codepen links are broken

File: README.md [hunk 2]
```diff
@@ -1,31 +1,38 @@
...
 
-## Example 2 (floating labels)
-[HTML](example-2/index.html) | [CSS](example-2/style.css) | [Javascript](example-2/script.js)
+## Running locally
 
-![](screenshots/example-2.gif)
+We recommend to serve the examples via a webserver. Although its plain HTML, CSS and JavaScript Mollie Components needs a valid hostename (e.g. localhost or 127.0.0.1). For this readme we use the [serve](https://www.npmjs.com/package/serve) package.
```
Comment (pabloroman) [inline README.md]: Is this recommended, or is a must? I think the first sentence contradicts the second.

File: README.md (hunk 1 above)
Comment (Marinolinderhof) [reply README.md]: Yes all of them are "fake" but you're totally right. We can't merge yet. This should wait till we have all the examples in place. 

I think it makes sense to create a feature branch. I will do that

File: README.md [hunk 3]
```diff
@@ -1,31 +1,38 @@
...
+- [Styling](https://docs.mollie.com/guides/mollie-components/styling)
 
-## Example 1 (regular labels)
-[HTML](example-1/index.html) | [CSS](example-1/style.css) | [Javascript](example-1/script.js)
+## Examples
 
-![](screenshots/example-1.gif)
+- [Example 1](./example-1) Bare minimum example ([codepen](example-1))
+- [Example 2](./example-2) Floating labels ([codepen](example-2))
+- [Example 3](./example-3) Compact version ([codepen](example-3))
```
Comment (el-angel) [inline README.md:16]: should we have a ticket that makes sure that in the end this is edited to the correct links and the amount of actual examples in this repository?

**Review by el-angel — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Marinolinderhof) [reply README.md:16]: yes, you're right. It should be one of the acceptance criteria of the examples

**Pull request merged**